tests/test_feature_selection.py

```diff
@@ -0,0 +1,80 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
+import pandas as pd
+
+from feature_engine.feature_selection import FeatureEliminator
+
```

Comment:
could you change the format of the tests so that each test case is actually a separate test? see for example this file:
https://github.com/solegalli/feature_engine/blob/master/tests/test_wrappers.py

I will be reformatting all the tests to this, because it is much simple to know which test is falling if they are individual. Each test should have a name describing that it is testing.

Reply:
Absolutely, I kind of thought along the same lines but didn't wanted to mess with the existing structure of tests. I will reformat them.